tests/x11/libreoffice/libreoffice_mainmenu_components.pm

```diff
@@ -95,16 +94,14 @@ sub run {
     $self->open_overview();
     type_string "base";
     # tag is base-install means libreoffice-base not installed
-    if (assert_screen 'base-install') {
+    if (check_screen 'base-install') {
```

Comment:
I'm not sure how stable will it be with 0 timeout (which is default), I would rather use `assert_screen` on multiple tags and then use `match_has_tag` to enable conditional behavior.

Reply:
Yes, I also think it's better. But, This "if(){..}" is only happened on TW, so I choose to use tags rather than "is_opensuse" as conditional behavior. As you suggest, I should make a needle on SLE too. But, I have not make a pull request to SLE and "qam-regression-installation" always test failed(terminated once over 2 hours) because network limited [#804]. (http://10.67.19.181/tests/804).